### ISSUE TITLE
Video quality & show name

### ISSUE THREAD

**Issue by dbukannan** [chatby/Shahid.bundle#1]
Is it possible to set a specific video quality i.e. 720p?
Show name is missing only an id is available

**Comment by chatby** [comment]
I'm only getting so far two options. The first is 'rtmpe' with 360p quality and second, with 'm3u8' and it has better quality. I noticed that m3u8 doesn't work with most of computers. so, I chose the low quality for the computers and high quality with devices such as iOS, Android and Roku which I prefer the most. About the Show name, Plex doesn't display the Arabic letters. So, I don't think it supports yet. I will still work to find how to get a better quality when using the computer browser.

**Comment by dbukannan** [comment]
Yes I agree with you. After checking it from an iOS device I guess 360p is good enough. But from the computer browser, I think 480p should be the minimum quality.
